File: gradle/packages/cli-function/src/main/kotlin/com/prog/cli/function/Random.kt
package com.prog.cli.function

import com.prog.numbers.utils.operations.FactorialUtils.factorialRandom
import com.prog.numbers.utils.operations.FibonacciUtils.fibonacciRandom

object Random {
    @JvmStatic
    fun main(args: Array<String>) {
        println("===================================================")
        printRandomNumbers()
        println("===================================================")
    }

    fun printRandomNumbers() {
        println(fibonacciRandom())
        println(factorialRandom())
    }
}

File: dotnet/src/CLI.Function/Random/Random.cs
using Mathematics = Prog.Numbers.Utils.Operations;

namespace Prog.CLI.Function;

public class Random
{
    public static void PrintRandomNumbers()
    {
        Console.WriteLine("Some random numbers :");
        Console.WriteLine(Mathematics.FibonacciUtils.FibonacciRandom());
        Console.WriteLine(Mathematics.FactorialUtils.FactorialRandom());
    }
}
